src/main/webapp/about.jsp

```diff
@@ -58,20 +59,40 @@
       <p> <br>
         <strong>Meet the Team!</strong>
         <ul>
+            <table cellpadding = "10" cellspacing = "1">
+              <tr><td align="center"><img src="About-IMG/Default-Profile-IMG.png" width = "150" height = "150"><br>
+                <a href="https://github.com/MarcelRG" target="_blank"><i style="color: #000000" class="fa fa-github" aria-hidden="true"></i></a>
 
-        		 <img src="Default-Profile-IMG.png" width = "150" height = "150" style = "float:left">
-        		<li> <strong>Marcel Gonzalez</strong>
-        		Currently studying Computer Science at Tecnologico de Monterrey </li> <br> <br> <br>
+                <a href="ADD LINK HERE!!!!" target="_blank"><i style="color: #000000" class="fa fa-linkedin-square" aria-hidden="true"></i></a>
 
+                <br><br></td>
+                <td><strong>Marcel Gonzalez</strong>
+                Currently studying Computer Science at Tecnologico de Monterrey<br><br><br></td></tr>
```

Comment:
Since you are now using tables to separate text, do you need all the <br> tags here and below? It seems like you should be using margin or padding CSS to separate the rows instead of blank lines.